packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -789,6 +794,13 @@ export default class TradeStore extends BaseStore {
 
     async onChange(e: { target: { name: string; value: unknown } }) {
         const { name, value } = e.target;
+        if (
+            name == 'contract_type' &&
+            ['accumulator', 'match_diff', 'even_odd', 'over_under'].includes(value as string)
```

Comment:
where are you handling a case when contract_type has tick and user has chosen tick as timeframe?

Reply:
Also, are we disabling/hiding the indicator if its not available?

Reply:
When the hasPredictionIndicators flag is true in trade-chart.tsx (useEffect) . It shows that either of the above trade-type is selected and they have unsupported indicator as well . For timeperiod,  each of these trade-type has restricted 1 tick time interval .
  
  We delete the indicators on appearnace of the pop-up (save it in localStorage). And on Cancel/Continue button of popup, we decide whether to make them reappear and delete permanently.

Reply:
We don't need this condition here. There shouldn't be any problem setting `prev_contract_type` for all contract types.